homeassistant/components/nederlandse_spoorwegen/strings.json

```
@@ -0,0 +1,144 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Nederlandse Spoorwegen Setup",
+        "description": "Set up your Nederlandse Spoorwegen integration.",
+        "data": {
+          "api_key": "API key"
```

Can use a common string here:
```suggestion
          "api_key": "[%key:common::config_flow::data::api_key%]"
```